recipe/meta.yaml

```diff
@@ -31,7 +28,7 @@ requirements:
     - m2-patch            # [win]
   host:
     - python
-    - cython >=0.29.22
+    - cython >=3.0.7
     - freetds
     - pip
     - setuptools >=54.0
```

Comment:
Could you replace `toml` by `tomli`? See https://github.com/pymssql/pymssql/blob/v2.3.2/pyproject.toml#L7C5-L7C10